test/integration/smart_answer_flows/energy_grants_calculator_test.rb

```diff
@@ -55,6 +55,21 @@ class EnergyGrantsCalculatorTest < ActiveSupport::TestCase
             assert_current_node :outcome_help_with_bills # outcome 1
           end
         end
+
+        context "answer before 5th May 1953" do
+          setup do
+            add_response '01/05/1953'
+          end
+
+          should "be eligible for winter fuel payment" do
+            assert_equal :winter_fuel_payment, current_state.calculator.age_variant
+          end
```

Comment:
I'd remove this test as it's behaviour that's being unit tested in [energy_grants_calculator_test](https://github.com/alphagov/smart-answers/blob/master/test/unit/calculators/energy_grants_calculator_test.rb#L22). That test should be updated in this commit to reflect the new behaviour.

I'd change the existing unit test from:

```ruby
should 'return :winter_fuel_payment if date of birth before 05-07-1951' do
  @calculator.date_of_birth = Date.new(1951, 7, 5) - 1
  assert_equal :winter_fuel_payment, @calculator.age_variant
end
```

To something like:

```ruby
should 'return :winter_fuel_payment if date of birth before winter_fuel_payment_threshold' do
  @calculator.stubs(:winter_fuel_payment_threshold).returns(Date.today)
  @calculator.date_of_birth = Date.yesterday
  assert_equal :winter_fuel_payment, @calculator.age_variant
end
```